cmd/arc_summary/arc_summary.py

```diff
@@ -705,8 +705,8 @@ def _l2arc_summary(Kstat):
         )
         sys.stdout.write("\n")
 
-        if arc["l2_arc_evicts"]['lock_retries'] + \
-                arc["l2_arc_evicts"]["reading"] > 0:
+        if arc["l2_arc_evicts"]['lock_retries'] != '0' or \
+           arc["l2_arc_evicts"]["reading"] != '0':
```

Comment:
Yeah, doing an int() conversion was my first local fix, only to realize that this string value comes from fHits() which returns it's string formatted with a 'k', 'm', 'b', ... which would make int() throw. A seemingly obvious alternative would be to make the dictionary value be the integer value and move the call to fHits() to where the value is being displayed to humans.